batch-add-recording-relationships.user.js

```diff
@@ -1374,4 +1397,10 @@ function batch_recording_rels() {
     function rowTitleCell($row) {
         return $row.children(`td:has(${TITLE_SELECTOR})`);
     }
+
+    function build_edit_note(edit_mode) {
+        return `${document.getElementById('bpr-edit-note').value.trim()}\n\n\n'''${gm_info.script.name}''' ${
```

Comment:
Doesn't it need an ensured blank line before EM DASH?

Please, tell me if [with blank line](https://test.musicbrainz.org/edit/100280061) doesn't look better than [without blank line](https://test.musicbrainz.org/edit/100280060). :)

Reply:
Personally I prefer notes without an additional blank line before the separator, but that might be because I rarely write edit notes which contain empty lines. And I kind of doubt that someone will write long edit notes which require empty lines for this script, the most relevant information which I would have put into it is already being added by the userscript itself :grin:

Reply:
> All of the MB userscripts of which I know seem to use a line with a dash to separate their signature from the body of the edit note, including your own

I was about to change my own scripts to use that two-blank-lines before signature, that I found more clean.
But OK for this script, it's really not up to me.
I will go with your suggestion, exactly. :+1: